dmca-comments.css

```diff
@@ -0,0 +1 @@
x auto 0}h1.page-title{font-size:18px;color:#d0021b;text-align:center;text-transform:uppercase}h2.sub-title{text-align:center;font-size:26px;margin-bottom:48px}input[type=checkbox]{display:none}input[type=checkbox]+label:before{font-family:FontAwesome;display:inline-block;font-size:30px;color:#d0021b}label{font-size:22px;cursor:pointer;cursor:hand}input[type=checkbox]+label:before{content:"\f10c"}input[type=checkbox]+label:before{letter-spacing:30px}input[type=checkbox]:checked+label:before{content:"\f058"}input[type=checkbox]:checked+label:before{letter-spacing:30px}.page{margin:85px 0}.callout{color:#d0021b;font-size:26px;margin:48px;text-align:center}address,p{font-size:18px}address{color:#555;margin:36px 0;padding:0 0 0 12px;border-left:5px solid #ddd}.btn{font-size:18px;text-transform:uppercase;border-radius:48px;padding:11px 22px;background:#000;color:#fff}.btn.top-action{padding:11px 22px 10px 22px}.btn:active,.btn:hover{background:#000;color:#fff}.btn.transparent{border:2px solid #d0021b;color:#d0021b;background:transparent}.btn.twitter{background-color:#55acee}.btn.facebook{background-color:#3b5998}.btn.share-button{margin:48px auto;margin-right:8px}.btn.share-button i{margin-left:8px}.btn.share-button.last{margin-right:0}.logo{margin:0 auto;width:100px;display:block}.description{font-size:8px;display:none}.exemption-categories{margin-bottom:120px}.exemption-group{margin:30px 0}.exemption-group p{margin-left:56px}.sticky-nav{position:fixed;bottom:0;text-align:center;padding:24px;width:100%;background-color:#fff;box-shadow:0 0 20px #888}.sticky-nav p{color:#d0021b;font-size:20px;font-family:'Montserrat',sans-serif;margin:0}.sticky-nav .btn{display:inline;margin-left:24px;font-size:18px}.sticky-nav .btn i{margin-left:8px}.action-nav{text-align:center;font-size:18px;margin:36px 0}.action-nav.top-action{margin-top:-25px}.accordion-row{position:relative;margin:12px 0}.accordion-container{cursor:pointer}.accordion-title{font-weight:bold}.accordion-description{margin-left:56px}.exemption-label{color:#d0021b;font-size:16px;text-transform:uppercase;margin-bottom:4px;font-weight:bold}.accordion-slide{display:none;margin-left:56px}.accordion-slide textarea{width:100%}.links{margin-left:56px}.accordion-link{margin-left:6px}.letter,.exemptions,.contact{display:none}.error-message,.has-error{color:#d0021b}.success-page{text-align:center}.success-page .page p{margin:16px 0}.success-page .page img{width:100%}.success{font-size:48px;text-transform:uppercase;text-align:center}.footer{text-align:center;margin:48px 0}.logos{list-style-type:none;margin:48px auto;padding:0;overflow:hidden;width:946px}@media only screen and (max-width: 599px){.logos{width:100%}}.logos li{display:inline-block;float:left;margin:0 24px}@media only screen and (max-width: 599px){.logos li{margin:24px !important}}.logos li:first-of-type{margin-left:0}.logos li:last-of-type{margin-right:0}.logos img{max-width:150px}@media only screen and (max-width: 599px){.logos img{max-width:100%}}
```

Comment:
We don't need to commit this css.  The build and deploy process converts less -> css and minifies.